frontend/src/pages/AbsencePlanner.tsx

```diff
@@ -434,6 +544,41 @@ export const VacationPlanner = () => {
     </div>
   );
 
+  //The empty heading tags make the top border go all the way out
+  const getAbsenceHeadings = () => (
+    <tr>
+      <th>Start date</th>
+      <th>End date</th>
+      <th></th>
+      <th></th>
+    </tr>
+  );
+  const getAbsenceTable = tableData.map((element, index) => {
+    return (
+      <tr key={index.toString()}>
+        <td>{formatDate(element.startDate, dateFormat)}</td>
+        <td>{formatDate(element.endDate, dateFormat)}</td>
+        <td>
+          <img src={pencil} className="pencil-icon" alt="pencil to edit" />
+        </td>
+        <td>
+          <button
+            onClick={() => {
+              onRemoveEntriesButton(element.entryIds);
+            }}
+            className="trash-button"
+          >
+            <img
+              src={trash}
+              className="trash-icon"
+              alt="trash icon to delete"
+            />
+          </button>
+        </td>
+      </tr>
+    );
+  });
+
```

Comment:
This is a little odd to me. Both these functions look like reusable React components that should live in their own files. You included them into this file, but still let them be separate functions instead of having them linearly in the JSX-code below. Is there any specific reason you did it this way? To me, it's very confusing.